x-pack/plugins/infra/public/pages/metrics/metrics_explorer/components/metrics.tsx

```diff
@@ -26,7 +26,7 @@ interface SelectedOption {
 }
 
 export const MetricsExplorerMetrics = ({ options, onChange, fields, autoFocus = false }: Props) => {
-  const colors = Object.keys(MetricsExplorerColor) as MetricsExplorerColor[];
+  const colors = Object.keys(Color) as Color[];
```

Comment:
There was a mistake here in the casting. It should have been:

```ts
const colors = Object.keys(Color) as Array<keyof typeof Color>;
```

it works because the enum keys and its values are the same, but if they ever change it will break

```ts
enum Color {
  color0 = "COLOR0",
  color1 = "COLOR1",
  ...
}

type T1 = Color[]; // "COLOR0" | "COLOR1" ... (well, not _really_, but it's the closest representation)
type T2 = keyof typeof Color // "color0" | "color1" ...
```